src/components/empty-search/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { motion } from 'framer-motion';
+import { PropsWithChildren } from 'react';
+import { Image } from '@atb/components/icon';
+
```

Comment:
Any logic behind separation of imports?

Reply:
Copy paste issue. I like to separate css from the others.

Reply:
I'm on board with that.